packages/next/src/shared/lib/normalized-asset-prefix.ts

```diff
@@ -0,0 +1,16 @@
+export function normalizedAssetPrefix(assetPrefix: string | undefined): string {
+  const escapedAssetPrefix = assetPrefix?.replace(/^\/+/, '') || false
+
+  // assetPrefix as a url
+  if (escapedAssetPrefix && escapedAssetPrefix.startsWith('http')) {
```

Comment:
```suggestion
  if (escapedAssetPrefix && escapedAssetPrefix.includes('://')) {
```
Let's check for the pattern we're actually splitting on